layouts/partials/footer.html

```diff
@@ -3,8 +3,14 @@
 				<div class="row">
 					<div class="col-xs-12 text-xs-center">
 						<p>
-							Copyright &copy; 2016 <a href="{{ .Site.Params.website }}">{{ .Site.Params.author }}</a>. All rights reserved. Made with <i class="fa fa-heart" aria-hidden="true"></i> and <a href="https://gohugo.io">Hugo</a>. Hosted and sponsored by <a href="https://www.digitalocean.com/">DigitalOcean</a>.
+							Copyright &copy; {{ .Now.Year }} <a href="{{ .Site.Params.website }}">{{ .Site.Params.author }}</a>. All rights reserved. Made with <i class="fa fa-heart" aria-hidden="true"></i> and <a href="https://gohugo.io">Hugo</a>. Hosted and sponsored by <a href="https://www.digitalocean.com/">DigitalOcean</a>.
```

Comment:
`Copyright &copy; 2016{{ .neq .Now.Year 2016 }}-{{ .Now.Year }}{{ end }}` maybe? 

Which would result in `Copyright © 2016` until new year when it goes `Copyright © 2016-2017` :)